scripts/install-helm.sh

```diff
@@ -21,6 +21,27 @@ NODE_LABEL_ZONE="test.failure-domain/zone"
 REGION_VALUE="testregion"
 ZONE_VALUE="testzone"
 
+example() {
```

Comment:
other functions use the `function` keyword. keep coding style the same for this file please

Reply:
The majority of functions in this script don't make use of the function keyword. Should I get rid of the `function` keyword since only 2 functions make use of it?

Reply:
Added a commit to make sure the convention is the same throughout the file.